### ISSUE TITLE
empty: support positive len

### ISSUE THREAD

**Issue by ccoVeille** [Antonboom/testifylint#139]
this pattern is common

```go
var arr []string
assert.True(t, 0 < len(arr))
```

https://github.com/search?q=language%3Ago+%22assert.True%28t%2C+0+%3C+len%28%22&type=code

`compares` checker will fix it to

```go
var arr []string
assert.Positive(t, len(l))
```

But that's it, while we should convert it to `Empty`

```go
assert.NotEmpty(t, l)
```

It's a new use case

_Originally posted by @ccoVeille in https://github.com/Antonboom/testifylint/issues/119#issuecomment-2181545452_

**Comment by ccoVeille** [comment]
Please confirm if the logic is valid for you, then I can code it

**Comment by ccoVeille** [comment]
Well this one could be added to #119 maybe, not sure if we have to do it at the same time or separately

**Comment by Antonboom** [comment]
> But this one won't be detected

Do you check it before spam issue?

Duplicate of https://github.com/Antonboom/testifylint/issues/122
Implemented in https://github.com/Antonboom/testifylint/pull/124
